waka_waka/Scenes/main.tscn

```diff
@@ -0,0 +1,18 @@
+[gd_scene load_steps=4 format=4 uid="uid://tnjwrwydx71v"]
+
+[ext_resource type="PackedScene" uid="uid://c4ev5q5xc3lts" path="res://Scenes/Player.tscn" id="1_sl6g6"]
+[ext_resource type="TileSet" uid="uid://2kbf0ykmvv3r" path="res://Assets/maze_spritemap.tres" id="2_5prns"]
+[ext_resource type="Script" path="res://Scenes/maze.gd" id="3_frwhm"]
+
+[node name="Main" type="Node2D"]
+
+[node name="Player" parent="." instance=ExtResource("1_sl6g6")]
+
+[node name="Camera2D" type="Camera2D" parent="Player"]
```

Comment:
Oh, I haven't messed with camera at all up to this point. Is that only in order to follow the player?

Reply:
Yep => A child node inherits its parent's attributes unless overridden, so making one Node2D follow another can be done just by making it the child of the other.

If I understand it correctly, nodes "inherit" properties in two ways:
- Through the class hierarchy, like object-oriented inheritance. Static and defined by class structure. For properties and functions that all instances of the class should have. Directly accessible in code.
- Through their hierarchy in the scene tree. Child nodes can access and interact with parent node's properties by reference. `get_parent()` returns a Node's current parent in the scene tree. Also `remove_child()` and `add_child()`: caveat: no signal/event is triggered automatically.